Pipfile

```diff
@@ -15,10 +15,11 @@ redisgraph = ">=1.7,<1.9"
 
 # Linters and other development tools
 bandit = ">=1.5.1,<2"
+black = ">=19.10b0"  # This is still marked as a beta release, make sure to run "pipenv lock --pre".
```

Comment:
Since the pre-release isn't optional here, I'll follow the official pipenv suggestion at [this link](https://pipenv-fork.readthedocs.io/en/latest/diagnose.html#your-dependencies-could-not-be-resolved) and generically tell pipenv to always allow pre-releases.